10.9-libcxx/stable/main/finkinfo/libs/pythonmods/setuptools-scm-py.info

```diff
@@ -28,10 +28,17 @@ CompileScript: <<
 
 # pytest now depends on this package so we can't run tests anymore
 # without a circular dependency.
-#InfoTest: <<
-#   TestDepends: pytest-py%type_pkg[python], mercurial, git
-#   TestScript: PYTHONPATH=. %p/bin/py.test-%type_raw[python] testing || exit 2
-#<<
+InfoTest: <<
+    TestDepends: mercurial, git
```

Comment:
It probably wants this [lz4-py](https://pypi.org/project/lz4/0.9.0), that's a fairly dated version. And then tries subsequently to install `setuptools==0.9.8` and `setuptools==12.0.1`, so I don't see how to satisfy this with a package dependency, even if I tried.

Reply:
From Debian, changing the test command to this skips the download tests:
````
TESTS_TO_EXCLUDE="not setuptools_support and not test_pip_download"
TESTS_TO_EXCLUDE="$TESTS_TO_EXCLUDE and not test_pkginfo_noscmroot and not test_use_scm_version_callable"
PYTHONPATH=. %p/bin/python%type_raw[python] -m pytest -k "$TESTS_TO_EXCLUDE"
````

(but I think the 2nd set of exclusions can be skipped for us)
Results before:

````python
plugins: hypothesis-3.65.1
collected 102 items                                                                                                 

testing/test_basic_api.py .s........                                                      [  9%]
testing/test_file_finder.py ..............................                                [ 39%]
testing/test_functions.py .................                                               [ 55%]
testing/test_git.py ..............                                                        [ 69%]
testing/test_main.py .                                                                    [ 70%]
testing/test_mercurial.py ...............                                                 [ 85%]
testing/test_regressions.py ..F.s                                                         [ 90%]
testing/test_setuptools_support.py FFF                                                    [ 93%]
testing/test_version.py .......                                                           [100%]
````
After
````python
plugins: hypothesis-3.65.1
collected 102 items / 6 deselected                                                                                  

testing/test_basic_api.py .s........                                                       [ 10%]
testing/test_file_finder.py ..............................                                 [ 41%]
testing/test_functions.py .................                                                [ 59%]
testing/test_git.py ..............                                                         [ 73%]
testing/test_main.py .                                                                     [ 75%]
testing/test_mercurial.py ...............                                                  [ 90%]
testing/test_regressions.py .s                                                             [ 92%]
testing/test_version.py .......                                                            [100%]
````

Reply:
If updating to 3.0.2 will take some work, let's work on fixing our current 2.1.0 here, and then updating to 3.0.2 since nothing else in the current tree is requiring 3.0.2.

Reply:
It's rather 2.1.0 causing troubles. 3.0 works by itself, and making 1.15 a `BuildConflict` lets `fink -m update` progress smoothly. So I prefer the latter, although I am not 100 % comfortable not knowing why 2.1.0 passes on your system, but not for me...